app/utils/CheckAction.scala

```diff
@@ -105,4 +106,23 @@ class Base58CheckAction(parser: BodyParsers.Default, address: String) extends Ac
 
 class Base58CheckActionFactory @Inject()(parser: BodyParsers.Default) {
   def apply(address: String): Base58CheckAction = new Base58CheckAction(parser, address)
+}
+
+class DateFromToCheckAction(parser: BodyParsers.Default, fromDate: String, toDate: String) extends ActionBuilderImpl(parser) {
+  private val sdf: SimpleDateFormat = new SimpleDateFormat("yyyy-MM-dd hh:mm:ss")
+  sdf.setLenient(true)
+
+  override def invokeBlock[A](request: Request[A], block: Request[A] => Future[Result]): Future[Result] = {
+    val fromDateParse: Try[Date] = Try(sdf.parse(fromDate + " 0:0:0"))
+    val toDateParse: Try[Date] = Try(sdf.parse(toDate + " 23:59:59"))
+    if (fromDateParse.map(from => toDateParse.map(_.getTime >= from.getTime).getOrElse(false)).getOrElse(false))
```

Comment:
You can change condition to `fromDateParse.flatMap(from => toDateParse.map(_.getTime >= from.getTime)).getOrElse(false)`